lib/dnsimple/client/tlds.rb

```diff
@@ -42,6 +42,38 @@ def all_tlds(options = {})
         paginate(:tlds, options)
       end
       alias :all :all_tlds
+
+      # Gets a TLD's details
+      #
+      # @see https://developer.dnsimple.com/v2/tlds/#get
+      #
+      # @param  tld [#to_s] The TLD name.
+      # @param  options [Hash]
+      # @return [Dnsimple::Response<Dnsimple::Struct::Tld>]
+      #
+      # @raise  [Dnsimple::NotFoundError]
```

Comment:
@weppos should we document this everywhere else? I've seen several endpoints that could also raise a `Dnsimple::NotFoundError` but didn't document it. I thought it was because it was a subclass of `Dnsimple::RequestError` but I may be wrong.